tfe/resource_tfe_agent_pool.go

```diff
@@ -0,0 +1,107 @@
+package tfe
+
+import (
+	"fmt"
+	"log"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceTFEAgentPool() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceTFEAgentPoolCreate,
+		Read:   resourceTFEAgentPoolRead,
+		Update: resourceTFEAgentPoolUpdate,
+		Delete: resourceTFEAgentPoolDelete,
```

Comment:
what do you think about adding the ability to import agent pools? since we didn't have provider support the first time around?

Reply:
Yes, absolutely. Wasn't a concern months ago when I wrote this so I totally forgot, let's do it!